### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by Isaactoo** [js-platform/p2p#21]
Your project here looks remarkable, and I'd like to use it, but I cannot find the license for it. Please add a license file. Maybe put it under an MIT License? 

Thanks :)  ,
Isaac

**Comment by modeswitch** [comment]
@luser Any issue for you with putting this under MIT?

**Comment by luser** [comment]
Nope. I was going to ask you this same thing, and I sort of assumed it was MIT given that you put that license header on the copy of the client code in the Emscripten repo.
